verde/base/base_classes.py

```diff
@@ -246,18 +252,20 @@ def grid(
         Parameters
         ----------
         region : list = [W, E, S, N]
-            The boundaries of a given region in Cartesian or geographic
-            coordinates.
+            The boundaries of a given region.
         shape : tuple = (n_north, n_east) or None
             The number of points in the South-North and West-East directions,
             respectively.
         spacing : tuple = (s_north, s_east) or None
             The grid spacing in the South-North and West-East directions,
             respectively.
         dims : list or None
-            The names of the northing and easting data dimensions,
-            respectively, in the output grid. Defaults to ``['northing',
-            'easting']``. **NOTE: This is an exception to the "easting" then
+            The names of the northing/latitude and easting/longitude data
+            dimensions, respectively, in the output grid. Defaults are set
+            though ``dims`` class variable. Dims must be defined in the
+            following order: ``["northing", "easting"]`` or
+            ``["latitude", "longitude"]``.
+            **NOTE: This is an exception to the "easting" then
```

Comment:
```suggestion
            The names of the northing/latitude and easting/longitude data
            dimensions, respectively, in the output grid. Default is determined 
            from the ``dims`` attribute of the class. Must be defined in the
            following order: northing dimension, easting dimension.
            **NOTE: This is an exception to the "easting" then
```

I don't think we need to add "longitude" and "latitude" everywhere. That is bound to get confusing and hard to keep consistent. I expect users know that easting => longitude and northing => latitude (which is why I like those names as generic alternatives).